examples/tensorflow-mnist-model-inversion/src/tasks.py

```diff
@@ -14,6 +14,7 @@
 #
 # ACCESS THE FULL CC BY 4.0 LICENSE HERE:
 # https://creativecommons.org/licenses/by/4.0/legalcode
+
```

Comment:
This file https://github.com/usnistgov/dioptra/blob/888d19880458e020b25a0f356f1913649b2b6e25/examples/tensorflow-mnist-model-inversion/src/tasks.py can be deleted, since [task-plugins/securingai_custom/evaluation/tensorflow.py](https://github.com/usnistgov/dioptra/blob/888d19880458e020b25a0f356f1913649b2b6e25/examples/tensorflow-mnist-model-inversion/task-plugins/securingai_custom/evaluation/tensorflow.py) contains the same functions.